xml/System/String.xml

```diff
@@ -10576,7 +10591,7 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
  The following example creates a comma separated value list by substituting commas for the blanks between a series of numbers.  
   
  [!code-cpp[string.replace1#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.replace1/CPP/string.replace1.cpp#1)]
```

Comment:
the sample for Replace(String, String) should also be interactive

Reply:
Done in https://github.com/dotnet/samples/pull/1171

Reply:
Done in https://github.com/dotnet/samples/pull/1171